config-connector/solutions/projects/kpt/project-hierarchy/README.md

```diff
@@ -0,0 +1,63 @@
+Project Hierarchy
+==================================================
+
+# NAME
+
+  project-hierarchy
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a folder in an organization, and a project
+  beneath it. Make sure your Config Connector
+  service account has the folder creator and
+  project creator role for your organization.
+
+  In order to use, replace the
+  `${BILLING_ACCOUNT_ID?}` and `${ORG_ID?}` values
+  with your billing account and organization ID.
+  You will need to replace the project ID, since a
+  project with the given ID already exists.
+
+
+  Unfortunately, to create a project under a
```

Comment:
Not sure if we want to say "Unfortunately, ..." Sounds discouraging to me.

Reply:
Yes, I'll reword it.